app/services/decisions/case_decision_tree.rb

```diff
@@ -128,8 +128,6 @@ def blank_date_required?
     end
 
     def submission_root_step
```

Comment:
No longer need this helper method. Can just put `edit('/steps/submission/more_information')` back into the `def after_ioj` helper